cmd/cosign/errors/generate_docs.go

```diff
@@ -0,0 +1,89 @@
+package errors
+
+import (
+	"fmt"
+	"go/ast"
+	godoc "go/doc"
+	"go/parser"
+	"go/token"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// GenerateExitCodeDocs will generate documentation for the exit codes
+// that the cosign application will throw. Inspired by elemental-cli who
+// convieniently solved this problem prior: https://github.com/rancher/elemental-cli/blob/main/docs/generate_docs.go
+// there was no need to change (apart from some file paths) due to it
+// giving us exactly what we want without error.
+func GenerateExitCodeDocs() error {
```

Comment:
Damn, beautiful code. Only a genius could write something as beautiful as this ;)

Reply:
Standing on the shoulders of giants as they say 😝